packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -401,35 +389,35 @@ export default class TableColumnResizeEditing extends Plugin {
 
 		// At this point we change only the editor view - we don't want other users to see our changes yet,
 		// so we can't apply them in the model.
-		editingView.change( writer => this._applyResizingAttributesToTable( writer, target, viewTable ) );
+		editingView.change( writer => this._applyResizingAttributesToTable( writer, viewTable ) );
 	}
 
 	/**
-	 * Calculate the dom column widths. It is done by taking the width of the widest cell
-	 * from each table column (we rely on the TableWalker to determine
-	 * which column the cell belongs to).
+	 * Calculates the dom column widths. It is done by taking the width of the widest cell
+	 * from each table column (we rely on the  {@link module:table/tablewalker~TableWalker}
+	 * to determine which column the cell belongs to).
 	 *
 	 * @private
 	 * @param {module:engine/model/element~Element} modelTable A table which columns should be measured.
-	 * @returns {Array.<Number>} Widths expressed in pixels (without unit).
+	 * @returns {Array.<Number>} Column widths expressed in pixels (without unit).
```

Comment:
```suggestion
	 * @returns {Array.<Number>} Columns’ widths expressed in pixels (without unit).
```

Not entirely sure if that's the best way to refer to widths of multiple columns - but that's what Bartek B suggested me and it sounds legit (though a little more complex) 🙂